### PR TITLE
Content-Length variable we use for sending update file needs to be case insensitive

### PR DESCRIPTION
**Pull request by bugobliterator**
* also skip overriding SERIAL_ORDER in CubePilot-PPPGW

### CODE REVIEW

File: Tools/AP_Bootloader/network.cpp [hunk 1]
```diff
@@ -484,11 +484,20 @@ void BL_Network::handle_request(SocketAPM *sock)
...
 
     if (strncmp(headers, "POST / ", 7) == 0) {
-        const char *clen = "\r\nContent-Length:";
-        const char *p = strstr(headers, clen);
-        if (p != nullptr) {
-            p += strlen(clen);
-            const uint32_t content_length = atoi(p);
+        const char *clen1 = "\r\nContent-Length:";
+        const char *clen2 = "\r\ncontent-length:";
+        const char *p1 = strstr(headers, clen1);
```
Comment (tridge) [inline Tools/AP_Bootloader/network.cpp:489]: char *p = strstr(headers, clen);
if (p == nullptr) {
 p = strstr(headers, clen2);
}

**Comment by tridge** [comment]
should remove the ECC change

**Comment by peterbarker** [comment]
> * also skip overriding SERIAL_ORDER in CubePilot-PPPGW

It's really strange to put this in the same PR.

**Comment by bugobliterator** [comment]
> > * also skip overriding SERIAL_ORDER in CubePilot-PPPGW
> 
> It's really strange to put this in the same PR.

Otherwise will need to separate builds of Bootloaders.

**Review by tridge — APPROVED** [review]

**Pull request merged**